tests/e2e/logging/test_openshift-logging.py

```diff
@@ -0,0 +1,148 @@
+    )
+    assert sc_obj, f"Failed to create storage class"
+    assert obj.create_instance_in_clusterlogging(sc_name=sc_obj.name)
+
+    # Check the health of the cluster-logging
+    assert obj.check_health_of_clusterlogging()
+
+
+def teardown(sc_obj, cbp_obj):
+    """
+    The teardown will uninstall the openshift-logging from the cluster
+    """
+    sc_obj.delete()
+    cbp_obj.delete()
+    uninstall_cluster_logging()
+
+
+@pytest.mark.usefixtures(
+    test_fixture.__name__
+)
+class TestLogging_in_EFK_stack(E2ETest):
+    """
+    The class contains the testcases related to openshift-logging
+    """
+    @pytest.fixture()
+    def create_pvc_and_deploymentconfig_pod(self, request, pvc_factory):
+        """
+
+        """
+        def finalizer():
+            helpers.delete_deploymentconfig(pod_obj)
+
+        request.addfinalizer(finalizer)
+
+        # Create pvc
+        pvc_obj = pvc_factory()
+
+        # Create service_account to get privilege for deployment pods
+        sa_name = helpers.create_serviceaccount(pvc_obj.project.namespace)
+
+        helpers.add_scc_policy(sa_name=sa_name.name, namespace=pvc_obj.project.namespace)
+
+        pod_obj = helpers.create_pod(
+            interface_type=constants.CEPHBLOCKPOOL,
+            pvc_name=pvc_obj.name,
+            namespace=pvc_obj.project.namespace,
+            sa_name=sa_name.name,
+            dc_deployment=True
+        )
+        helpers.wait_for_resource_state(resource=pod_obj, state=constants.STATUS_RUNNING)
+        return pod_obj, pvc_obj
+
+    @pytest.mark.polarion_id("OCS-657")
+    @tier1
+    @retry(ModuleNotFoundError, 6, 300, 3)
+    def test_create_new_project_to_verify_logging(self, create_pvc_and_deploymentconfig_pod):
+        """
+        This function creates new project to verify logging in EFK stack
+        1. Creates new project
+        2. Creates PVC
+        3. Creates Deployment pod in the new_project and run-io on the app pod
+        4. Logs into the EFK stack and check for new_project
+        5. And checks for the file_count in the new_project in EFK stack
+        """
+
+        pod_obj, pvc_obj = create_pvc_and_deploymentconfig_pod
+
+        # Running IO on the app_pod
+        pod_obj.run_io(storage_type='fs', size=8000)
+
+        # Searching for new_project in EFK stack
+        pod_list = get_all_pods(namespace='openshift-logging')
+        elasticsearch_pod = [
+            pod.name for pod in pod_list if pod.name.startswith('elasticsearch')
+        ]
+        elasticsearch_pod_obj = get_pod_obj(
+            name=elasticsearch_pod[1], namespace='openshift-logging'
+        )
+        projects = elasticsearch_pod_obj.exec_cmd_on_pod(command='indices | grep project', out_yaml_format=True)
+        logger.info(projects)
+        if pvc_obj.project.namespace in projects:
+            logger.info("The new project exists in the EFK stack")
+            file_count = elasticsearch_pod_obj.exec_cmd_on_pod(
```

Comment:
what is the purpose of capturing this value?

Reply:
if the new-project is available then collect the log file count in it. It tells the shard count and number of files. If you feel this check is not necessary we can remove this check.
```{'count': 2, '_shards': {'total': 3, 'successful': 3, 'skipped': 0, 'failed': 0}}```

Reply:
@Shrivaibavi okay, we simply collect this value and don't interpret anything out of it. I don't see any value in it. It would be good if we write more logs and see if this count value increases. 
 Ideally, we should be looking for the log message in kibana ui/api. But until we figure out a way to do that via automation, let's use this validation.

Reply:
Right now I have removed that check. Probably I can try injecting log messages when i start making use of openshift-svt.

Reply:
@keesturam is this not yet resolved for you?